### PR TITLE
Fixes #23769 - maintain consistency in documentation for foreman/katello

### PR DESCRIPTION
**Pull request by rabajaj0509**

### CODE REVIEW

**Review by dLobatog — APPROVED** [review]
Thanks @rahulbajaj0509, I'll merge if tests pass

**Comment by ares** [comment]
@dLobatog please don't forget to set redmine release after merging, setting to 1.19 now

**Pull request merged**
